readme.md

```diff
@@ -197,6 +214,13 @@ Default: `false`
 
 Suppress output produced by the PHP-server.
 
+### settings
```

Comment:
Not a big fan of the option name. Maybe `directives`, `iniDirectives`, `phpDirectives`, ... ?